layouts/partials/post_preview.html

```diff
@@ -20,11 +20,14 @@ <h3 class="post-subtitle">
         {{ partial "post_meta.html" . }}
     </p>
     <div class="post-entry">
-        {{ if .Truncated }}
-        {{ .Summary }}
-        <a href="{{ .Permalink }}" class="post-read-more">[{{ i18n "readMore" }}]</a>
+        {{ if .Summary }}
+            {{ .Summary }}
```

Comment:
Hey there! Thanks for the PR, this indeed fixed the issue also for me. But when defining a summary, now the [Read More] link is missing, so I suggest to add it here as well:

```suggestion
            {{ .Summary }}
            <a href="{{ .Permalink }}" class="post-read-more">[{{ i18n "readMore" }}]</a>
```